### PR TITLE
feat(googleMapsPlatform): create connector

### PR DESCRIPTION
**Pull request by Oleksiivanov**
## Description

Google Maps Platform Connector.
Allows : 
- validate address
- format address
- get place id for the address
- get postal address from (postal code, region code, etc)
- calculate the distance and duration between two addresses 

https://user-images.githubusercontent.com/108869886/227038028-6038938f-9488-4c2c-8979-207b5db16a9e.mov

## Related issues
https://github.com/camunda/team-connectors/issues/363

### CODE REVIEW

File: connectors/google-maps-platform/element-templates/google-maps-platform-connector.json [hunk 1]
```diff
@@ -0,0 +1,533 @@
+{
+  "$schema": "https://unpkg.com/@camunda/zeebe-element-templates-json-schema/resources/schema.json",
+  "name": "Google Maps Platform Connector",
+  "id": "io.camunda.connectors.GoogleMapsPlatform.v1",
+  "description": "Orchestrate your Google Maps Platform operations with Camunda. You can validate and format address, get postal address, calculate distance, etc.",
```
Comment (bastiankoerber) [inline connectors/google-maps-platform/element-templates/google-maps-platform-connector.json:5]: orhcestrate doesn't fit in this contect, add this as description:
The Google Maps connector validates addresses, retrieves postal addresses and calculates distances.

File: connectors/google-maps-platform/element-templates/google-maps-platform-connector.json [hunk 2]
```diff
@@ -0,0 +1,533 @@
...
+        {
+          "name": "Validate an address",
+          "value": "validateAddress"
+        },
+        {
+          "name": "Get place ID",
+          "value": "getPlaceID"
+        },
+        {
+          "name": "Directions",
```
Comment (bastiankoerber) [inline connectors/google-maps-platform/element-templates/google-maps-platform-connector.json:66]: what do you mean with directtions?

Do you mean distance? then call it: Calculate Distance

File: connectors/google-maps-platform/element-templates/google-maps-platform-connector.json [hunk 3]
```diff
@@ -0,0 +1,533 @@
...
+      }
+    },
+    {
+      "label": "Operation type",
+      "id": "operationType",
+      "group": "operation",
+      "type": "Dropdown",
+      "choices": [
+        {
+          "name": "Validate an address",
```
Comment (bastiankoerber) [inline connectors/google-maps-platform/element-templates/google-maps-platform-connector.json:58]: replace with Validate Adress

File: connectors/google-maps-platform/element-templates/google-maps-platform-connector.json [hunk 4]
```diff
@@ -0,0 +1,533 @@
...
+        "property": "operationType",
+        "oneOf": [
+          "getPlaceID",
+          "directions"
+        ]
+      }
+    },
+    {
+      "label": "API Key",
+      "description": "Google Maps Platform API key. Details in the <a href='https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/microsoft-teams/'>connector documentation</a>",
```
Comment (bastiankoerber) [inline connectors/google-maps-platform/element-templates/google-maps-platform-connector.json:116]: replace the documentation link

File: connectors/google-maps-platform/element-templates/google-maps-platform-connector.json [hunk 5]
```diff
@@ -0,0 +1,533 @@
...
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "validateAddress"
+        ]
+      }
+    },
+    {
+      "label": "Address",
+      "description": "Complete address. Detailes in a <a href='https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/google-maps-platform/'> connector documantation</a>",
```
Comment (bastiankoerber) [inline connectors/google-maps-platform/element-templates/google-maps-platform-connector.json:194]: typo in details

File: connectors/google-maps-platform/element-templates/google-maps-platform-connector.json [hunk 6]
```diff
@@ -0,0 +1,533 @@
...
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "getPlaceID"
+        ]
+      }
+    },
+    {
+      "label": "Destination",
+      "description": "The place ID value to which you wish to calculate directions",
```
Comment (bastiankoerber) [inline connectors/google-maps-platform/element-templates/google-maps-platform-connector.json:215]: replace with: The place ID value that you want to use as the destination for calculating distance.

File: connectors/google-maps-platform/element-templates/google-maps-platform-connector.json [hunk 7]
```diff
@@ -0,0 +1,533 @@
...
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "directions"
+        ]
+      }
+    },
+    {
+      "label": "Origin",
+      "description": "The place ID value from which you wish to calculate directions.",
```
Comment (bastiankoerber) [inline connectors/google-maps-platform/element-templates/google-maps-platform-connector.json:236]: replace with: The place ID value that you want to use as the starting point for calculating distance.

File: connectors/google-maps-platform/element-templates/google-maps-platform-connector.json [hunk 8]
```diff
@@ -0,0 +1,533 @@
...
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "directions"
+        ]
+      }
+    },
+    {
+      "label": "Mode",
+      "description": "For the calculation of distances and directions, you may specify the transportation mode to use",
```
Comment (bastiankoerber) [inline connectors/google-maps-platform/element-templates/google-maps-platform-connector.json:284]: replace with: You can specify the transportation mode to use when calculating distances and directions.

**Review by bastiankoerber — CHANGES_REQUESTED** [review]
some wording and one wrong link. Rest looks good :-)

File: connectors/google-maps-platform/element-templates/google-maps-platform-connector.json [hunk 9]
```diff
@@ -0,0 +1,533 @@
...
+          "value": "calculateDistance"
+        }
+      ],
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operationType"
+      }
+    },
+    {
+      "description": "post method",
```
Comment (igpetrov) [inline connectors/google-maps-platform/element-templates/google-maps-platform-connector.json:76]: Normalize or remove (applicable to all similar occurances)

File: connectors/google-maps-platform/element-templates/google-maps-platform-connector.json [hunk 10]
```diff
@@ -0,0 +1,533 @@
...
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "getPlaceID",
+          "calculateDistance"
+        ]
+      }
+    },
+    {
+      "label": "API Key",
```
Comment (igpetrov) [inline connectors/google-maps-platform/element-templates/google-maps-platform-connector.json:115]:
```suggestion
      "label": "API key",
```

**Review by igpetrov — CHANGES_REQUESTED** [review]
LGTM. Minor changes requested.

**Review by igpetrov — APPROVED** [review]

**Comment by igpetrov** [comment]
Don't forget to squash the PR into 1 commit

**Pull request merged**